app/assets/javascripts/utilities.coffee

```diff
@@ -14,4 +14,8 @@ $.fn.addDialog = (partial, title) ->
     title: title
     open: -> $(this).html(partial)
     buttons:
-      Cancel: -> $(this).dialog("close")
\ No newline at end of file
+      Cancel: -> $(this).dialog("close")
+
+this.removeAllLabwares = removeAllLabwares = (id) -> 
+  alert("remove all labwares for" + id)
+  $("#remove-all-labwares-form").addDialog("<%= escape_javascript(render('remove_all_labwares')) %>", "Remove all Labware")
```

Comment:
How is this form #remove-all-labwares-form, or this function removeAllLabwares,  connected with the functionality?

Reply:
It isn't. I was going to do it with Javascript but it was more hassle than it was worth. I will remove it.